pyke/lightcurve.py

```diff
@@ -93,37 +93,36 @@ def flatten(self, window_length=101, polyorder=3, **kwargs):
     def fold(self, phase, period):
         return LightCurve(((self.time - phase + 0.5 * period) / period) % 1 - 0.5, self.flux)
 
-    def draw(self):
-        raise NotImplementedError("Should we implement a LightCurveDrawer class?")
-
     def to_csv(self):
         raise NotImplementedError()
 
-    def plot(self, ax=None, norm=True,**kwargs):
+    def plot(self, t=None, flux=None, flux_err=None, ax=None, norm=True, xlabel = 'Time - 2454833 (days)',
+            ylabel = 'Normalised Flux',color='#363636', fill=True, grid=True, **kwargs):
         """
         Plot a Light Curve.
         """
         if ax is None:
             fig, ax = plt.subplots(1)
-        t0=float(int(self.time[0] / 100) * 100.0)
-        f = self.flux
-        ferr = self.flux_err
+        if t == None:
+            t = self.time
+        if flux == None:
+            flux = self.flux
+        if flux_err == None:
+            flux_err = self.flux_err
         if norm:
-            ferr/=np.nanmedian(f)
-            f/=np.nanmedian(f)
-        try:
-            plt.errorbar(self.time-t0, f,ferr, color='#363636', **kwargs)
-        except:
-            plt.errorbar(self.time-t0, f, ferr, **kwargs)
+            flux_err/=np.nanmedian(flux)
```

Comment:
I believe the correct formula should be ``flux_err/=np.sqrt(np.nanmedian(flux))``?